### DISCUSSION TITLE
How to Change the Background Color of a Select Component in Daisy UI ?

### DISCUSSION THREAD

**Discussion by Foshati** [saadeghi/daisyui#3137] [Q&A] [answered]
Hello and respect,

How can I change the color of this Select component in Daisy UI, or is there a method to change the background color of this part?
<img width="454" alt="Screenshot 2024-07-18 at 10 45 01 PM" src="https://github.com/user-attachments/assets/ca099797-29db-4e4e-b431-2267b4822476">

**Comment by saadeghi** [comment, marked as answer]
Unfortunately CSS can't change the colors of the menu of `<select>` element. Each OS/browser has their own style for this part and it's also different on desktop/mobile.

There are JS libraries which simulate a `<select>` element using divs, and with those libraries it is possible to change the colors.

For example you can use Headless UI and then you can add daisyUI colors or any other style to it.

Let me know if you have any questions
